operator/v1/types_network.go

```diff
@@ -100,12 +100,33 @@ type DefaultNetworkDefinition struct {
 	KuryrConfig *KuryrConfig `json:"kuryrConfig,omitempty"`
 }
 
+// SimpleMacvlanConfig contains configurations for macvlan interface.
+type SimpleMacvlanConfig struct {
+	// master is the host interface to create the macvlan interface from.
+	// If not specified, it will be default route interface
+	// +optional
+	Master string `json:"master,omitempty"`
+
+	// ipam indicates which IPAM module will be used for IP Address Management (IPAM)
+	// now supports 'dhcp'. If it's not specified does it default to dhcp.
```

Comment:
This text is phrased as as question. Is it dhcp by default or not?  If it's only allowed to be dhcp, why is it a configuration option here?

Reply:
Currently (4.2) we have 'dhcp', however, we are planning to have alternative for ipam (like static) near future (4.3 might be), hence we have it.

Reply:
we're hoping to add more ipam providers in the coming releases. If you want us to drop this field now, we can.

Reply:
@deads2k should we keep or drop the field for now?

Reply:
If you're planning not to use it for now - drop it. It's easier to add than change/remove existing field.

Reply:
Sure. I've removed it.